pyproject.toml

```diff
@@ -82,47 +87,30 @@ linux-kerberos = [
   "requests-kerberos"
 ]
 
-test = [
-  "pytest",
-  "pytest-cov",
-  "uvicorn",
-  "fastapi",
-  "pydantic",
-  "requests-mock",
-  "pytest-mock",
-  "covertable",
-  "asgi_gssapi"
-]
-
-doc = [
-  "pyansys-sphinx-theme",
-  "numpydoc",
-  "sphinx",
-  "sphinx_autodoc_typehints",
-  "sphinx-notfound-page",
-  "sphinx-copybutton"
-]
-
 [tool.tox]
 legacy_tox_ini = """
 [tox]
-envlist = py37,py38,py39,py310
+envlist = py37,py38,py39,py310,py311
```

Comment:
We should do this in BoM Analytics as well if we haven't already.

Reply:
It's already done there, only need to turn it on in the strategy matrix and it should test!